content/news/2025-07-28-collab-clermont/index.md

```diff
@@ -0,0 +1,20 @@
+---
+title: 2025 Fink Collaboration Meeting
+date: 2025-07-28
+cardimage: lpca2.jpeg
+---
+The 2025 Fink Collaboration Meeting happened from 16 to 18 July 2025, in Clermont Ferrand - France
+<!--more-->
+
+Part of the Fink community met in Clermont Ferrand, during a few hot summer days, to discuss the future of Fink and make preparations for the upcoming Rubin data.  
+
+The three days were full with long discussion sessions, heated debates, a few presentations and a lot of dreams and expectations for the near future. 
+
+We also had the opportunity to experiment a little of local culture and cuisine with a visit to the [Chateaux de Murol](https://www.murolchateau.com/).
```

Comment:
[...] with a visit to the [Chateaux de Murol](https://www.murolchateau.com/), a castle originally built in the 11th century.